src/renderer/components/table/table.tsx

```diff
@@ -62,13 +63,32 @@ export class Table extends React.Component<TableProps> {
     sortSyncWithUrl: true,
   };
 
-  @observable sortParams: Partial<TableSortParams> = Object.assign(
-    this.props.sortSyncWithUrl ? {
-      sortBy: sortByUrlParam.get(),
-      orderBy: orderByUrlParam.get(),
-    } : {},
-    this.props.sortByDefault,
-  );
+  constructor(props: TableProps) {
+    super(props);
+    this.initSortParams();
+  }
+
+  initSortParams() {
+    const prevSortParams = getSortParams(this.props.tableId);
+
+    if (!prevSortParams) {
+      setSortParams(this.props.tableId, Object.assign(
+        this.props.sortSyncWithUrl ? {
+          sortBy: sortByUrlParam.get(),
+          orderBy: orderByUrlParam.get(),
+        } : {},
+        this.props.sortByDefault,
+      ));
+    }
+  }
```

Comment:
Most probably we don't need to `sortSyncWithUrl` since params is persistent.
Something like this should be enough:

```suggestion
@computed get sortParams(){
this.props.sortByDefault
 return Object.assign({}, this.props.sortByDefault, getSortParams(this.props.tableId));
}
```

@Nokel81 @aleksfront wdyt?

Reply:
Yeah that is probably the better way to do it.

Reply:
thank you, fixed